quickstarts/velero-prometheus/config.yml

```diff
@@ -0,0 +1,65 @@
+
+slug: velero-prometheus
+
+title: Velero (Prometheus)
+
+description: |+
+  ## A comprehensive Velero monitoring system
+
+  Velero is a tool used to backup and restore Kubernetes cluster resources and persistent volumes.
+
+  New Relic provides a Velero Quickstart which allows you to monitor your Velero schedules and jobs with ease.
+
+  ### New Relic - a perfect tool to monitor Velero using Prometheus.
+
+  Velero has become an important part of Kubernetes architectures, providing features to ensure continuity of state during disaster recover, and ease the burden of systems migration. 
+
+  Using the [New Relic Prometheus Remote-Write](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/) integration you can deliver your critical Velero metrics from your internally supported instance of Prometheus directly to New Relic where the rest of your observability data resides.
+
+  ### New Relic Velero quickstart features
+
+  Our Velero Quickstart include out-of-the-box dashboards and alerts, including data such as:
+  - Overview Schedules (# successes, # failures) + charts with duration and sizes of backups 
+
+  ### Value of the Velero (Prometheus) quickstart
+
+  The Velero (Prometheus) Quickstart provides a visual snapshot of all the key health information related to your Velero clients and the Kubernetes Clusters they are helping to maintain. Monitoring is made easy via the clear, color-coded dashboard which showcases schedule execution and much more.
+
+summary: |
+  New Relic provides crucial tools for monitoring your Velero clients using the [Prometheus Remote-Write](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/) integration. It supports features such as a visual dashboard complete with charts detailing all critical performance and health metrics relevant to your Velero system.
```

Comment:
If it is possible, can we remove the link in this summary? Looking at the preview there seems to be some oddity with showing it as a link.

If you open the PR preview comment in this PR conversation you'll be able to see it!